### DISCUSSION TITLE
Is there a way to add an automatic year change in the copyright at the bottom?

### DISCUSSION THREAD

**Discussion by JEREDEK** [DSJAS/DSJAS#46] [Ideas]
I keep finding myself stressed over the copyright black mesa 2018 at the bottom of the page, it would be nice and i think quite easy to implement an automatic year count with PHP

**Comment by ejv2** [comment]
I spy this as an opportunity for a joke. We could make the copyright date 1776 or 1492 or something ridiculously like that. Or we could spell it in roman numerals. Or we could write it in Latin. There are a whole host of things we could do. Ideas appreciated.

**Reply by SinisterSpatula** [reply]
I'd prefer it always use the current year and go for realism.  Maybe could be a setting where you specify custom text and have replacement values like %date%

**Reply by JEREDEK** [reply]
Yeah, sometimes i need realism, but other times i like to joke around. A separate field in the settings would be much appreciated!

**Comment by ejv2** [comment]
I have placed this on the todo list and will add it shortly.
